### PR TITLE
Tidy up readme.md

### PR DESCRIPTION
**Pull request by hamishwillee**
This updates the project readme.md to good standard for release. It uses a similar structure to the readme's for dk-android and dk-python.

It is currently in draft, and will need updates after the following are known:
* [x] Is there a Gitter channel - what is URL?
* [x] Is there a support forum - what is URL?
* [x] There is a CI test - is that just linux - use the linux icon?
* [x] Are there windows/mac tests. Can we have shield/link for them?

A lot of information on how to create analysers and missing analyzers has been removed - there are now links to the official documentation on this.

### CODE REVIEW

**Comment by mrpollo** [comment]
Gitter: https://gitter.im/dronekit/dronekit-la

File: README.md [hunk 1]
```diff
@@ -1,79 +1,82 @@
-# DroneKit LA
+# DroneKit LogAnalyzer (LA)
```
Comment (mrpollo) [inline README.md]: I've been writing this the opposite way and with a space between Log Analyzer **DroneKit LA (Log Analyzer)**, I think it reads better and it differentiates itself further from the current **LogAnalyzer** in Ardupilot

File: README.md [hunk 2]
```diff
@@ -1,79 +1,82 @@
...
- - the attitude control tests should ignore periods where the craft is not armed
- - the brownout report should give relative, not absolute, height at end of flight
- 
-    
-
-
-// - two fundamental types of test
-//  - is the software working correctly (EKF issues)
-//  - is the vehicle doing sensible things (attitude etc)
+Copyright 2016 3D Robotics, Inc.
```
Comment (mrpollo) [inline README.md]: Actually I'm not so sure about this, can we leave it out by now?

Comment (hamishwillee) [inline README.md]: @mrpollo - you're suggesting we lose the copyright notice?

File: README.md (hunk 1 above)
Comment (hamishwillee) [inline README.md]: OK, I'll update that.

**Comment by hamishwillee** [comment]
Updated with gitter/support icons/links. Fixed title. Need clarity on:

* whether you "really" want to remove copyright notice
* what is the setup on mac to build dk-la.

This would be OK to go out live right now though.

**Comment by mrpollo** [comment]
Mac tests https://travis-ci.org/dronekit/dronekit-la

**Comment by mrpollo** [comment]
I updated the badges, they are all shields and they are all showing up good now, I'm going to proceed with merging

**Pull request merged**